nvbench/detail/measure_cold.cuh

```diff
@@ -97,6 +102,10 @@ protected:
   nvbench::float64_t m_skip_time{};
   nvbench::float64_t m_timeout{};
 
+  nvbench::float32_t m_throttle_threshold{0.75f};     // [% of peak SM clock rate]
```

Comment:
minor: No need to provide default values here -- they're always set from the `state`. This way there's less to keep synced with the docs (delay currently doesn't match docs).